src/utils/init/utils.js

```diff
@@ -186,7 +186,10 @@ const saveNetlifyToml = async ({ siteRoot, config, buildCmd, buildDir, functions
   }
 }
 
-const formatErrorMessage = ({ message, error }) => `${message} with error: ${chalk.red(error.message)}`
+const formatErrorMessage = ({ message, error }) => {
+  const errorMessage = error.json ? `${error.message} - ${JSON.stringify(error.json)}` : error.message
```

Comment:
GitHub returns a json object with more information about the error. The structure changes based on the error, but it's still useful to print it as is.